README.md

```diff
@@ -15,10 +15,26 @@
 [s6]: https://tokei.rs/b1/github/amethyst/laminar?category=code
 [s7]: https://codecov.io/gh/amethyst/laminar/branch/master/graphs/badge.svg
 
-This library implements some TCP-like features on top of an UDP-socket. 
-It will provide a lightweight message-based interface with certain guarantees like reliability, fragmentation congestion monitoring.
+Laminar is a semi-reliable UDP-based protocol for multiplayer games. This library implements wrappers around the UDP-protocol,
```

Comment:
`This library implements wrappers around a UDP socket`

Reply:
both are correct, but we are not wrapping around a socket perse but more around the protocol itself.

Reply:
well technically we are implementing a udp protocol on top of a udp socket protocol is already in udp via the p.

Reply:
Technically speaking David P. Reed implemented the UDP-protocol. This protocol was defined way back in 1980.

Reply:
![](https://media1.tenor.com/images/701a53f5ed5bb4df3af074b0e2c02ce5/tenor.gif?itemid=7556577)